bom/application/pom.xml

```diff
@@ -26,6 +26,7 @@
         <opentracing-concurrent.version>0.2.0</opentracing-concurrent.version>
         <opentracing-jdbc.version>0.0.12</opentracing-jdbc.version>
         <opentracing-kafka.version>0.1.13</opentracing-kafka.version>
+        <opentelemetry.version>0.16.0</opentelemetry.version>
```

Comment:
https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.0.0

Reply:
@jorsol Thanks. I've updated that locally, but haven't pushed updates to the PR yet